src/commcare_cloud/ansible/roles/nginx/vars/cchq_ssl.yml

```diff
@@ -67,7 +67,7 @@ nginx_sites:
       proxy_buffers: 8 64k
       proxy_buffer_size: 64k
       client_body_buffer_size: 512k
-    - name: '~ /a/[^/]+/(receiver|phone)/'
+    - name: '~ /a/[^/]+/(receiver|phone|admin/phone)/'
```

Comment:
Is this the right URL? I'm seeing `/hq/admin/phone/restore/`. Does the one you have here also work?

Reply:
Matches https://dimagi.atlassian.net/wiki/spaces/GS/pages/2144317344/Restore+File+and+Configuration+Options (internal docs) too.

Reply:
In the code I also see `/a/<domain>/phone/admin_restore/` here https://github.com/dimagi/commcare-hq/blob/44b56b86f8e750f9e0a816ef61e2565a185ce430/corehq/apps/ota/urls.py#L18-L19

Reply:
Oh yeah duh. Do we actually need both of those admin restore endpoints? I suppose the admin_restore endpoint has the domain readily available in a way that the hq/admin/phone/restore might not? I need to understand how a domain is chosen when not restoring a mobile worker since the domain isn't included in the username.

Reply:
I think it's also available to domain admins, whereas /hq/admin/... is only available to site admins (superusers)